contrib/systemd/btrbk.service.in

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=btrbk backup
 Documentation=man:btrbk(1)
+ConditionPathExists=|@CONFDIR@/btrbk/btrbk.conf
```

Comment:
I'm not using systemd myself, and my knowledge is a bit limited here (and no, not going to read through all the docs, all those options are one of the reasons for me not to use systemd).

This sounds like a good thing to have. does that mean that the service will not fail if the config file does not exist, even if you start it?

Reply:
Yes, AFAIU, it should not fail (that would require an `AssertPathExists`).